src/Gnugat/Medio/Model/Method.php

```diff
@@ -164,6 +169,11 @@ public function isStatic()
      */
     public function makeStatic()
     {
+        
+        if ($this->isAbstract()) {
+            throw new \Gnugat\Medio\Exception\DomainException("You can`t declare abstract method as static: " . $this->getName());
```

Comment:
You could add a use statement to avoid having to write the exception's fully qualified classname (we'd be able to see the whole message without scrolling, and by looking at the top of the file we'd be able to tell the dependencies).